x-pack/plugins/rule_registry/server/utils/create_lifecycle_executor.ts

```diff
@@ -241,7 +216,7 @@ export const createLifecycleExecutor =
         const currentAlertData = currentAlerts[alertId];
 
         if (!alertData) {
-          logger.warn(`Could not find alert data for ${alertId}`);
+          logger.debug(`[Rule Registry] Could not find alert data for ${alertId}`);
```

Comment:
Changed this to `debug` since it was emitting for every new alert and it's not really valuable for the end user.